cmd/internal/cli/inspect_linux.go

```diff
@@ -211,42 +166,94 @@ func getHelpCommand(appName string) string {
 	return getSingleFileCommand("runscript.help", "helpfile", appName)
 }
 
-func setAttribute(obj *inspectFormat, label string, value string) {
+func setAttribute(obj *inspectFormat, label, app string, value string) {
```

Comment:
```suggestion
func setAttribute(obj *inspectFormat, label string, app string, value string) {
```

Reply:
I personally don't think it is necessary.

Reply:
```suggestion
func setAttribute(obj *inspectFormat, label, app, value string) {
```